aws_lambda_powertools/utilities/parser/models/s3.py

```diff
@@ -11,88 +11,319 @@
 
 class S3EventRecordGlacierRestoreEventData(BaseModel):
     lifecycleRestorationExpiryTime: datetime
-    lifecycleRestoreStorageClass: str
+    lifecycleRestoreStorageClass: str = Field(
+        description="Source storage class for restore.",
+        examples=[
+            "standard",
+            "standard_ia",
+            "glacier",
+        ],
+    )
 
 
 class S3EventRecordGlacierEventData(BaseModel):
     restoreEventData: S3EventRecordGlacierRestoreEventData
```

Comment:
Can you add a description here? Please let me know if you need help writing them.